runtime/lua/vim/lsp/client.lua

```diff
@@ -867,12 +867,12 @@ end
 ---
 --- @param force? boolean
 function Client:stop(force)
-  if self:is_stopped() then
+  local rpc = self.rpc
+  if rpc.is_closing() then
     return
   end
 
   self._is_stopping = true
```

Comment:
With this change is `self._is_stopping` still useful? Or can we just rely on `rpc.is_closing()`?